test/spectests/spectests.cpp

```diff
@@ -485,7 +485,24 @@ class test_runner
         return fizzy::execute(*instance, *func_idx, args.data());
     }
 
-    bool check_integer_result(fizzy::Value actual_value, const json& expected)
+    bool check_i32_result(fizzy::Value actual_value, const json& expected)
```

Comment:
I think `read_integer_value` function has to be modified, too, because now it produces always 64-bit `Value`